src/main/java/org/prebid/server/spring/config/WebConfiguration.java

```diff
@@ -99,6 +94,7 @@
     private ExceptionHandler exceptionHandler;
 
     @Autowired
+    @Qualifier("router")
```

Comment:
Since qualifier is used here I would suggest making `router` bean name explicit in place of its declaration (`@Bean Router router(...)` method)